roles/sssd/templates/sssd.conf

```diff
@@ -70,8 +70,8 @@ min_id = {{ ldap_config.min_id }}
 max_id = {{ ldap_config.max_id }}
 enumerate = true
 cache_credentials = false
-ldap_tls_reqcert = demand
+ldap_tls_reqcert = {{ ldap_config.ldap_tls_reqcert | default('demand', true) }}
```

Comment:
In not in favor of making TLS for LDAP optional. Without TLS the connection is insecure and should fail.

Reply:
Playbooks were not working, and if were not able to solved it (as we now did), we would need to wait again before we could deploy the initial configuration. I added this as an option for test deployment, and made sure that it gets ignored by default. One would need to deliberately bypass it, by writing the variable.
I can also remove it from PR...

Reply:
Removed